include/irr/asset/filters/dithering/CWhiteNoiseDither.h

```diff
@@ -29,12 +32,28 @@ namespace irr
 
 				static float get(const state_type* state, const core::vectorSIMDu32& pixelCoord, const int32_t& channel)
 				{
-					// TODO: to define in future
+					auto getWangHash = [&]()
+					{
+						IImage::SBufferCopy::getLocalByteOffset(pixelCoord, bufferStridesHash);
+						uint32_t seed = IImage::SBufferCopy::getLocalByteOffset(pixelCoord, bufferStridesHash) * channel;
```

Comment:
dont use byte offsets, just compute a pixel index value `(((channel*extent.depth+z)*extent.height+y)*extent.width+x`

Reply:
what about it https://github.com/buildaworldnet/IrrlichtBAW/pull/521#issuecomment-683958930?

Reply:
corrected in following commit